staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -57,7 +57,19 @@ func NewGCEPersistentDiskCSITranslator() InTreePlugin {
 
 // TranslateInTreeStorageClassParametersToCSI translates InTree GCE storage class parameters to CSI storage class
 func (g *gcePersistentDiskCSITranslator) TranslateInTreeStorageClassParametersToCSI(scParameters map[string]string) (map[string]string, error) {
-	return scParameters, nil
+	np := map[string]string{}
+	// Strip out the deprecated "zone" and "zones" parameters since the CSI
+	// Driver does not accept these, the provisioner will translate this
+	// information into topologies for the CSI Driver in a seperate function
+
+	// TODO(#77235) Translate deprecated zone and zones labels to supported
+	// topologies in the external provisioner
```

Comment:
Will it make sense for the translation of zone/zones to be in the CSI translation library rather than external provisioner? Keeping it in the translation library may allow the external provisioner to only be aware of the modern topology and accessibility requirements.

I believe EBS will need it too @chengpan (and Azure as well as vspherevolume down the line as well).

Reply:
I just re-read the comment and realized you meant to say the translation will not be done in the external provisioner and it will only be aware of topologies. If that is the case, I think the above is already clarified.